src/libraries/System.Net.Security/tests/FunctionalTests/SslAuthenticationOptionsTest.cs

```diff
@@ -144,6 +144,32 @@ public async Task ClientOptions_TargetHostNull_OK()
                Assert.Equal(string.Empty, server.TargetHostName);
             }
         }
+
+        [Fact]
+        public void ClientOptions_ShallowCopy_OK()
+        {
+            using X509Certificate2 clientCert = Configuration.Certificates.GetClientCertificate();
+
+            var clientOptions = new SslClientAuthenticationOptions
+            {
+                AllowRenegotiation = false,
+                AllowTlsResume = false,
+                ApplicationProtocols = new List<SslApplicationProtocol> { SslApplicationProtocol.Http11, SslApplicationProtocol.Http2 },
+                CertificateRevocationCheckMode = X509RevocationMode.Online,
+                ClientCertificates = new X509CertificateCollection() { clientCert },
+                EnabledSslProtocols = SslProtocols.Tls12,
+                EncryptionPolicy = EncryptionPolicy.RequireEncryption,
+                TargetHost = "foo",
+                CertificateChainPolicy = new X509ChainPolicy(),
+                RemoteCertificateValidationCallback = new RemoteCertificateValidationCallback(delegate { return true; }),
+                LocalCertificateSelectionCallback = new LocalCertificateSelectionCallback(delegate { return null; }),
+                ClientCertificateContext = SslStreamCertificateContext.Create(clientCert, null, false),
```

Comment:
I assume all of these are non-default values?  Might be worth a comment mentioning that.